addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -330,8 +330,8 @@ def _process_step_forward_operator(self, discuss_channel):
 
         if discuss_channel.livechat_channel_id:
             human_operator = discuss_channel.livechat_channel_id._get_operator(
-                lang=discuss_channel.livechat_visitor_id.lang_id.code,
-                country_id=discuss_channel.country_id.id
+                lang=discuss_channel.livechat_visitor_id.lang_id.code if hasattr(discuss_channel, "livechat_visitor_id") else None,
```

Comment:
`livechat_visitor_id` is indeed only defined when the module `website_livechat` is installed, which requires to install `website` in addition to `im_livechat`: https://github.com/odoo/odoo/blob/17.0/addons/website_livechat/models/discuss_channel.py#L11